ember-cli-build.js

```diff
@@ -1,46 +1,47 @@
 /* global require, module */
 const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
-const yuidoc = require('broccoli-yuidoc');
+const Yuidoc = require('broccoli-yuidoc');
 const version = require('git-repo-version')();
-const Handlebars = require('handlebars');
 
-var sourceTrees = [];
+let sourceTrees = [];
 
 module.exports = function(defaults) {
-  var app = new EmberAddon(defaults, {
+  let app = new EmberAddon(defaults, {
     storeConfigInMeta: true,
     jscsOptions: {
       enabled: true,
-      testGenerator: function(relativePath, errors) {
+      testGenerator(relativePath, errors) {
         if (errors) {
-          errors = "\\n" + this.escapeErrorString(errors);
+          // eslint-disable-next-line prefer-template
```

Comment:
good question... actually all of that stuff can be removed anyways since we don't need JSCS anymore...